sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/kafka/BeamKafkaTableTest.java

```diff
@@ -184,24 +185,11 @@ private static Schema genSchema() {
             .build());
   }
 
-  private static class String2KvBytes extends DoFn<String, KV<byte[], byte[]>>
+  protected static class String2KvBytes extends DoFn<String, KV<byte[], byte[]>>
```

Comment:
It looks like this is only actually used in `BeamCsvTableTest`, let's just move it there rather than making it protected.

Reply:
It will also be used in JSON table provider, that's why it's here as protected. But I can move it in the json PR in order to keep this one more consistent.

Reply:
Ohh ok sorry about that. I'm fine with just making it protected here